node-util/sbin/oo-accept-node

```diff
@@ -852,6 +852,10 @@ def check_app_dirs
       user_fail(entry, "directory #{entry}/.env doesn't have OPENSHIFT_PRIMARY_CARTRIDGE_DIR")
     end
   end
+  # Bug 1281912 - Check for users that don't have a corresponding gear file
+  copy_all_users.each do |u|
+    user_fail(u, "user #{u} does not have a directory in #{$GEAR_BASE_DIR}")
```

Comment:
This doesn't look to match up with your example. `#{$GEAR_BASE_DIR}` will return `/var/lib/openshift`, not `/var/lib/openshift/<uuid>`, right? Your example also includes the word 'home', but it does not appear in the code.

Additionally, I think this error could be worded better. Perhaps something along the lines of:
`"Gear home directory #{$GEAR_BASE_DIR}/#{u} does not exist for user #{u}"`